projects/plugins/boost/app/assets/src/js/features/minify-meta/minify-meta.tsx

```diff
@@ -1,73 +1,91 @@
 import { useEffect, useState } from 'react';
 import { __, sprintf } from '@wordpress/i18n';
 import { type Props, useMetaQuery } from '$lib/stores/minify';
+import { recordBoostEvent } from '$lib/utils/analytics';
 import styles from './minify-meta.module.scss';
-import { Button } from '@automattic/jetpack-components';
-import Pencil from '$svg/pencil';
+import CollapsibleMeta from '$features/ui/collapsible-meta/collapsible-meta';
 
-const MetaComponent = ( { inputLabel, buttonText, placeholder, datasyncKey }: Props ) => {
+const MetaComponent = ( { buttonText, placeholder, datasyncKey }: Props ) => {
 	const [ values, updateValues ] = useMetaQuery( datasyncKey );
 	const [ inputValue, setInputValue ] = useState( () => values.join( ', ' ) );
-	const [ isEditing, setIsEditing ] = useState( false );
+
+	const concatenateType = datasyncKey === 'minify_js_excludes' ? 'js' : 'css';
+	const togglePanelTracksEvent = 'concatenate_' + concatenateType + '_panel_toggle'; // possible events: concatenate_js_panel_toggle, concatenate_css_panel_toggle
 
 	useEffect( () => {
 		setInputValue( values.join( ', ' ) );
 	}, [ values ] );
 
+	const onToggleHandler = ( isExpanded: boolean ) => {
+		! isExpanded && setInputValue( values.join( ', ' ) );
+	};
+
 	function save() {
+		/*
+		 * Possible Events:
+		 * concatenate_js_exceptions_save_clicked
+		 * concatenate_css_exceptions_save_clicked
+		 */
+		recordBoostEvent( 'concatenate_' + concatenateType + '_exceptions_save_clicked', {} );
+
 		updateValues( inputValue );
-		setIsEditing( false );
 	}
 
 	const htmlId = `jb-minify-meta-${ datasyncKey }`;
 
-	return (
-		<div className={ styles[ 'minify-meta' ] } data-testid={ `meta-${ datasyncKey }` }>
-			{ isEditing ? (
+	const summary =
+		values.length > 0
+			? /* Translators: %s refers to the list of excluded items. */
+			  sprintf( __( 'Except: %s', 'jetpack-boost' ), values.join( ', ' ) )
+			: '';
+
+	const content = (
+		<div className={ styles.body } data-testid={ `meta-${ datasyncKey }` }>
+			<div className={ styles.section }>
+				<div className={ styles.title }>{ __( 'Exceptions', 'jetpack-boost' ) }</div>
 				<div className={ styles[ 'manage-excludes' ] }>
-					<label htmlFor={ htmlId }>{ inputLabel }</label>
+					<span className={ styles[ 'sub-header' ] }>
+						{ sprintf(
+							/* Translators: %s refers to the type of script. */
+							__( 'Exclude the following %s scripts:', 'jetpack-boost' ),
+							datasyncKey === 'minify_js_excludes' ? 'JS' : 'CSS'
+						) }
+					</span>
 					<input
 						type="text"
 						value={ inputValue }
 						placeholder={ placeholder }
 						id={ htmlId }
 						onChange={ e => setInputValue( e.target.value ) }
+						onKeyDown={ e => {
+							if ( e.key === 'Enter' ) {
```

Comment:
I was going to suggest using `keyCode` as that use they actual code, since `key` returns different for the main enter and the numpad enter. You can check that out here - https://www.toptal.com/developers/keycode

However, it appears that `keyCode` has been deprecated.

I would only update this to support the numpad enter.

```suggestion
							if ( e.key === 'Enter' || e.key === 'NumpadEnter' ) {
```

Reply:
Thanks!